src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -354,3 +354,54 @@ void mpuGyroInit(gyroDev_t *gyro)
     UNUSED(gyro);
 #endif
 }
+
+uint8_t mpuGyroDLPF(gyroDev_t *gyro)
+{
+    uint8_t ret;
+    if (gyro->gyroRateKHz > GYRO_RATE_8_kHz) {
+        ret = 0;  // If gyro is in 32KHz mode then the DLPF bits aren't used - set to 0
+    } else {
+        switch (gyro->hardware_lpf) {
+            case GYRO_HARDWARE_LPF_NORMAL:
+                ret = 0;
```

Comment:
maybe (?) return directly from these switch cases?

Reply:
I would have except for that the entire switch is wrapped in a if/then/else that has to fall through anyway.  Otherwise have lots of returns which can end up being gotchas if people want to add code later.  Not a big deal to me either way though.  This is initialization code called once so performance isn't really a concern.

Reply:
Sounds good. Let's leave as-is.